SDK_versioned_docs/version-3.0.0/widgets/swap-widget-api.md

```diff
@@ -0,0 +1,50 @@
+---
+id: swap-widget-api
+slug: swap-widget/api
+title: Swap Widget API Reference
+sidebar_label: API Reference
+sidebar_position: 2
+---
+
+# Swap Widget API Reference
+
+## Required Parameters {#required-parameters}
+
+| Prop Name | Prop Type | Default Value | Description |
+| --- | --- | --- | --- |
+| `provider` | `any` | `undefined` | An [EIP-1193](https://eips.ethereum.org/EIPS/eip-1193) provider. |
```

Comment:
What about:
```
An [EIP-1193](https://eips.ethereum.org/EIPS/eip-1193) provider. This is required to swap.
```

Arguably this should be an optional parameter, because it is not required for display, only for swapping functionality. If we take this route, maybe we should just have a list of "Parameters".

Reply:
ok